draft-ietf-quic-qpack.md

```diff
@@ -1004,6 +1002,119 @@ Description:
+| 7     | etag                             |                                                       |
+| 8     | if-modified-since                |                                                       |
+| 9     | if-none-match                    |                                                       |
+| 10    | last-modified                    |                                                       |
+| 11    | link                             |                                                       |
+| 12    | location                         |                                                       |
+| 13    | referer                          |                                                       |
+| 14    | set-cookie                       |                                                       |
+| 15    | :method                          | CONNECT                                               |
+| 16    | :method                          | DELETE                                                |
+| 17    | :method                          | GET                                                   |
+| 18    | :method                          | HEAD                                                  |
+| 19    | :method                          | OPTIONS                                               |
+| 20    | :method                          | POST                                                  |
+| 21    | :method                          | PUT                                                   |
+| 22    | :scheme                          | http                                                  |
+| 23    | :scheme                          | https                                                 |
+| 24    | :status                          | 200                                                   |
+| 25    | :status                          | 304                                                   |
+| 26    | accept                           | */*                                                   |
+| 27    | accept                           | application/dns-message                               |
+| 28    | accept-encoding                  | gzip, deflate, br                                     |
+| 29    | accept-language                  |                                                       |
+| 30    | authorization                    |                                                       |
+| 31    | cache-control                    | max-age=0                                             |
+| 32    | cache-control                    | max-age=604800                                        |
+| 33    | cache-control                    | max-age=2592000                                       |
+| 34    | cache-control                    | no-cache                                              |
+| 35    | cache-control                    | no-cache, no-store, must-revalidate                   |
```

Comment:
this has equivalent semantics to 'no-store'. I know some variation on this is common practice, but it's bad practice, and I doubt that there's enough convergence on this particular spelling to make the codification of that bad practice worthwhile. Just do `no-store`.